etc/checkstyle.xml

```diff
@@ -73,7 +71,7 @@ under the License.
       <module name="RightCurly"/>
       <module name="RightCurly">
          <property name="option" value="alone"/>
-         <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
+         <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO, STATIC_INIT, INSTANCE_INIT"/>
```

Comment:
i just started going back through reverting the do-while changes and adding that check to replace the old config.